Cargo.toml

```diff
@@ -25,9 +25,7 @@ members = [
 # See more keys and their definitions at
 # https://doc.rust-lang.org/cargo/reference/manifest.html
 
-[dev-dependencies]
-iced = { version = "0.4", features = ["canvas"] }
-
 [dependencies]
-iced_native = "0.5"
-iced_graphics = { version = "0.3", features = ["canvas"] }
+iced = { version = "0.5.2", features = ["canvas"] }
```

Comment:
I can't get this to build with OctaSine unless `default-features = false` is set for iced and iced_audio features for wgpu and glow are added, like:

```
[features]
default = ["wgpu"]
glow = ["iced/glow"]
wgpu = ["iced/wgpu"]
```

Reply:
Oh, the canvas feature can definitely be kept! It is just that iced defaults to enabling wgpu, which makes widgets non-compatible with the glow backend. I don’t have the exact error message available now.

Reply:
Ok, I see. I'll update accordingly.